src/index.js

```diff
@@ -184,8 +185,11 @@ class CreditCardInput extends Component<Props, State> {
   };
 
   handleCardExpiryBlur = (e: SyntheticInputEvent<*>) => {
-    if (!payment.fns.validateCardExpiry(e.target.value)) {
-      this.setFieldInvalid('Expiry date is invalid');
+    let cardExpiryBlur = e.target.value.split(' / ').join('/');
+    if (creditCardExpiryValidator.isExpiryInvalid(cardExpiryBlur)) {
```

Comment:
Can we extract this fn into something like:

```javascript
const expiryError = creditCardExpiryValidator.isExpiryInvalid(cardExpiryBlur)
if (expiryError) {
  this.setFieldInvalid(expiryError);
}
```

Reply:
Fixed